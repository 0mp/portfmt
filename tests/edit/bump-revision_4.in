PORTNAME=	lastpass-cli
PORTVERSION=	1.3.1
DISTVERSIONPREFIX=	v
CATEGORIES=	security
