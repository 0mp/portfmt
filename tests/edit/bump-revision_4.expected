PORTNAME=	lastpass-cli
PORTVERSION=	1.3.1
DISTVERSIONPREFIX=	v
PORTREVISION=	1
CATEGORIES=	security
